profile/README.md

```diff
@@ -1,3 +1,43 @@
-# Helius ☀️
+<br>
 
-Our powerful APIs save you the headache of dealing with cryptic on-chain data, so you can focus on your product. It's time to build. 
+<p align="center">
+    <img width="300" src="/img/HeliusLogo.png" alt="Helius Logo">
+</p>
+
+<br>
+<h3 align="center">Solana's leading RPCs, APIs, Webhooks, and Data Infrastructure.&nbsp;</h3>
+
+
+<h3 align="center">
+    Our powerful APIs save you the headache of dealing with cryptic on-chain data, so you can focus on your product. It's time to build.
+</h3>
+
+
+<br>
+
+<p align="center">
+	<a href="https://discord.gg/helius"><img src="https://img.shields.io/discord/992616840251584633?label=discord&style=flat-square&color=5a66f6"></a>
+	&nbsp;
+    <a href="https://twitter.com/heliuslabs"><img src="https://img.shields.io/badge/twitter-follow_us-1d9bf0.svg?style=flat-square"></a>
+    &nbsp;
+    <a href="https://www.linkedin.com/company/heliusapi/"><img src="https://img.shields.io/badge/linkedin-connect_with_us-0a66c2.svg?style=flat-square"></a>
+    &nbsp;
+    <a href="mailto:info@helius.xyz"><img src="https://img.shields.io/badge/email-get_in_contact-blue?style=flat-square"></a>
+</p>
+
+<p align="center">
+	<a href="https://www.helius.dev/blog"><img height="25" src="/img/social/blog.svg" alt="Blog"></a>
```

Comment:
I believe the correct src should be `../img/social/blog.svg`. Same goes for all the image refs.

Reply:
I believe both would work as the`/img/social/blog.svg` path would just create an absolute path from the root directory which would still work with the current directory structure. Making it a relevant path would be a bit more flexible and I will update if you still think its a good idea.

Reply:
The absolute path should work in theory, but I'm trying to see how this markdown file would look by switching to the rich diff, and it seems the images aren't showing correctly.

Reply:
I've updated the file paths to relative paths but it still seems that the images don't show in the rich diff tab. They are however reflecting in my local fork.